bin/test.sh

```diff
@@ -20,7 +20,7 @@ set -euo pipefail
 echo ">>> wait for services"
 waitfor --verbose "http://${PUBSUB_EMULATOR_HOST}"
 waitfor --verbose "${STORAGE_EMULATOR_HOST}/storage/v1/b"
-waitfor --verbose --codes=200,404 "${SENTRY_DSN}"
+waitfor --verbose --codes={200,404} "${SENTRY_DSN}"
```

Comment:
I think I preferred the previous syntax of comma-separated codes, but requiring it to be specified multiple times and using this syntax seems fine.

Does this work in other non-bash shells?

Reply:
this works in many shells https://unix.stackexchange.com/a/92829

most notably it doesn't work in `/bin/sh`